src/custom/components/TransactionSettings/TransactionSettingsMod.tsx

```diff
@@ -294,8 +299,11 @@ export default function TransactionSettings({ placeholderSlippage }: Transaction
               color={theme.text1}
               text={
                 <Trans>
-                  Your swap expires and will not execute if it is pending for longer than the selected duration.
-                  {INPUT_OUTPUT_EXPLANATION}
+                  {isEthFlow
+                    ? `Native currency (e.g ETH) orders require a minimum transaction deadline threshold of ${
+                        DEADLINE_LOWER_THRESHOLD_SECONDS / 60
```

Comment:
Shouldn't this be rounded?

Reply:
Should be a multiple of 60, I think it's safe to not round it